contrib/buildrefactor/tests/python/pants_test/contrib/buildrefactor/buildozer_util.py

```diff
@@ -0,0 +1,32 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import fnmatch
+import os
+
+
+def clean_build_file(build_file):
```

Comment:
`clean_build_file` and `clean_build_directory` are probably unnecessary. You can following the example of creating a temp dir with auto cleanup then create BUILD file under the temp dir
https://github.com/pantsbuild/pants/blob/a09ac81420589749c3adc73e9fb02a0d8bc8447d/tests/python/pants_test/backend/jvm/tasks/test_scope_runtime_integration.py#L119-L148